includes/Dashboard.php

```diff
@@ -347,38 +347,31 @@ public function get_dashboard_settings() {
 		$settings = [
 			'id'         => 'web-stories-dashboard',
 			'config'     => [
-				'isRTL'               => is_rtl(),
-				'locale'              => ( new Locale() )->get_locale_settings(),
-				'newStoryURL'         => $new_story_url,
-				'editStoryURL'        => $edit_story_url,
-				'wpListURL'           => $classic_wp_list_url,
-				'assetsURL'           => trailingslashit( WEBSTORIES_ASSETS_URL ),
-				'cdnURL'              => trailingslashit( WEBSTORIES_CDN_URL ),
-				'version'             => WEBSTORIES_VERSION,
-				'encodeMarkup'        => $this->decoder->supports_decoding(),
-				'api'                 => [
+				'isRTL'              => is_rtl(),
+				'locale'             => ( new Locale() )->get_locale_settings(),
+				'newStoryURL'        => $new_story_url,
+				'editStoryURL'       => $edit_story_url,
+				'wpListURL'          => $classic_wp_list_url,
+				'assetsURL'          => trailingslashit( WEBSTORIES_ASSETS_URL ),
+				'cdnURL'             => trailingslashit( WEBSTORIES_CDN_URL ),
+				'version'            => WEBSTORIES_VERSION,
+				'encodeMarkup'       => $this->decoder->supports_decoding(),
+				'api'                => [
 					'stories'     => sprintf( '/web-stories/v1/%s/', $rest_base ),
 					'media'       => '/web-stories/v1/media/',
 					'currentUser' => '/web-stories/v1/users/me/',
 					'users'       => '/web-stories/v1/users/',
 					'templates'   => '/web-stories/v1/web-story-template/',
 					'settings'    => '/web-stories/v1/settings/',
 				],
-				'maxUpload'           => $max_upload_size,
-				'maxUploadFormatted'  => size_format( $max_upload_size ),
-				'capabilities'        => [
+				'maxUpload'          => $max_upload_size,
+				'maxUploadFormatted' => size_format( $max_upload_size ),
+				'capabilities'       => [
 					'canManageSettings'   => current_user_can( 'manage_options' ),
 					'canUploadFiles'      => current_user_can( 'upload_files' ),
 					'canReadPrivatePosts' => $can_read_private_posts,
 				],
-				'siteKitCapabilities' => [
-					'siteKitInstalled'      => $this->is_site_kit_plugin_installed(),
-					'siteKitActive'         => defined( 'GOOGLESITEKIT_VERSION' ),
-					'analyticsModuleActive' => false, // TODO: copy the logic from Analytics.php we need to share somehow.
-					'canInstallPlugins'     => current_user_can( 'install_plugins' ),
-					'canActivatePlugins'    => current_user_can( 'activate_plugin', 'google-site-kit/google-site-kit.php' ),
```

Comment:
Why were the capabilities check removed here? This seems like a regression over an improvement.

Reply:
It's unneeded code.